packages/packagers/xml/src/XMLPackager.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import type {Bundle, BundleGraph, NamedBundle} from '@parcel/types';
+import assert from 'assert';
+import {Packager} from '@parcel/plugin';
+import {
+  blobToString,
+  replaceInlineReferences,
+  replaceURLReferences,
+} from '@parcel/utils';
+import {DOMParser, XMLSerializer} from '@xmldom/xmldom';
+
+export default (new Packager({
+  async package({bundle, bundleGraph, getInlineBundleContents}) {
+    const assets = [];
+    bundle.traverseAssets(asset => {
+      assets.push(asset);
+    });
+
+    assert.strictEqual(
+      assets.length,
+      1,
+      'XML bundles must only contain one asset',
+    );
+
+    let asset = assets[0];
+    let code = await asset.getCode();
+    let parser = new DOMParser();
+    let dom = parser.parseFromString(code);
+
+    let inlineElements = dom.getElementsByTagNameNS(
+      'https://v2.parceljs.org',
+      'inline',
+    );
```

Comment:
I guess that should that be `publicURL`?

Reply:
No that's just an XML namespace. Ends up like `<inline xmlns="https://v2.parceljs.org" key="abc" />`, and that gets replaced during packaging. Just avoids conflicts with any other `<inline>` elements that may exist.